src/main/java/it/pagopa/pn/deliverypush/dto/timeline/details/CancelledDetailsInt.java

```diff
@@ -15,11 +16,16 @@
 @Getter
 @Setter
 @ToString
-public class CancelledDetailsInt implements RecipientRelatedTimelineElementDetails {
+public class CancelledDetailsInt implements TimelineElementDetailsInt {
 
-    private int recIndex;
+    private int notificationCost;
```

Comment:
nello schema vedevo che era "nullabile", quindi forse meglio mettere Integer?
          description: costo notifica in euro cents, può essere nullo se la notifica si è perfezionata prima per decorrenza termini

Reply:
non può essere null, ho corretto la descrizione copia e incolla